presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -268,6 +268,7 @@
     public static final String NATIVE_ORDER_BY_SPILL_MEMORY_THRESHOLD = "order_by_spill_memory_threshold";
     public static final String NATIVE_EXECUTION_ENABLED = "native_execution_enabled";
     public static final String NATIVE_EXECUTION_EXECUTABLE_PATH = "native_execution_executable_path";
+    public static final String NATIVE_EXECUTION_PROCESS_REUSE_ENABLED = "native_execution_process_reuse_enabled";
```

Comment:
I wonder if it is possible to always share native process within a single Spark executor and use different native processes when running multiple Spark executors in a test process.

Reply:
But is there a way to achieve the same without introducing a flag? Is it possible to have a single process per executor and have multiple processes if there are multiple executors?

Reply:
> But is there a way to achieve the same without introducing a flag? Is it possible to have a single process per executor and have multiple processes if there are multiple executors?

I believe in unit test since we have to use the [Spark local](https://books.japila.pl/apache-spark-internals/local/) scheduler to run the spark job, it'll run everything in a single JVM process (including driver/executor), so we can't achieve multiple executors in this mode but can only increase the # of threads to have more parallelism.